docker/7.1.0/ig/Dockerfile

```diff
@@ -3,6 +3,9 @@ FROM gcr.io/forgerock-io/ig:7.1.0
 # Copy all config files into the docker image.
 # The default ig directory is /var/ig, and it expects subfolders config/ and scripts/ (if required)
 
+ENV JAVA_OPTS="-Dorg.forgerock.http.TrustTransactionHeader=true"
```

Comment:
Is this flag needed for IG?

I believe the flag allows IG to trust an `x-forgerock-transaction-id` header passed in, but for our use case the TPP App should never be supplying this, instead they should send the `x-fapi-interaction-id`

Other FR apps won't response with this header, and IG is the entrypoint into other FR apps, therefore I don't see the use case.

Reply:
Fixed